cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -42,14 +42,12 @@ pub async fn create_swap<D, A: ActorState>(
     // construct a generator that watches alpha and beta ledger concurrently
     let mut generator = Gen::new({
         let dependencies = dependencies.clone();
-        |co| {
-            async move {
-                future::try_join(
-                    watch_alpha_ledger(&dependencies, &co, swap.alpha_htlc_params()),
-                    watch_beta_ledger(&dependencies, &co, swap.beta_htlc_params()),
-                )
-                .await
-            }
+        |co| async move {
+            future::try_join(
+                watch_alpha_ledger(&dependencies, &co, swap.alpha_htlc_params()),
+                watch_beta_ledger(&dependencies, &co, swap.beta_htlc_params()),
+            )
+            .await
```

Comment:
Is this random change an example of a `cargo fmt` that didn't get staged previously? :)

Reply:
No this is from updating rustfmt to a version that has the `--files-with-diff` option.